src/azure-cli/azure/cli/__main__.py

```diff
@@ -31,6 +31,12 @@
 
 logger = get_logger(__name__)
 
+ALPINE_WARNING_MESSAGE = (
+    "Azure CLI v2.63 is the latest version available on Alpine 10 and will not receive updates. "
+    "Consider migrating to the Azure Linux based image for Azure CLI. "
+    "For more information: https://docs.microsoft.com/..."
+)
```

Comment:
All GitHub action user will see this warning. We'd better to add the instruction to hide the warning.
The actions also need to migrate to Azure Linux based image in the future.
@MoChilia

Reply:
`az config set core.only_show_errors=true` can control this message.

Reply:
It's great to use `az config` to hide this warning message, but it may also hide other warnings. I will first migrate azure/cli action to use Mariner-based image to avoid this warning and ensure the latest azure-cli is available in azure/cli action. Later we may migrate to Azure Linux based image once it's ready.

Reply:
> It's great to use `az config` to hide this warning message, but it may also hide other warnings.

I think this is acceptable. I don't want to spend too much effort on this warning message as Alpine-based images will no longer be released/maintained.

Reply:
@jiasli, isn't it just `Alpine` and not `Alpine 10`?

Reply:
There is no “alpine 10”.

PS: There is Alpine 3.10. But we’re using 3.20 now. I think we can omit the version number in warning message.